README.md

```diff
@@ -4,6 +4,7 @@
 > Tooling to support the migration of UI5 projects by adapting code for new UI5 framework versions.
 
 [![Build Status](https://dev.azure.com/sap/opensource/_apis/build/status/SAP.ui5-migration?branchName=master)](https://dev.azure.com/sap/opensource/_build/latest?definitionId=41&branchName=master)
+[![npm version](https://badge.fury.io/js/%40ui5%2Fmigration.svg)](https://badge.fury.io/js/%40ui5%2Fmigration)
```

Comment:
The link can point directly to https://www.npmjs.com/package/@ui5/migration. One less redirect = faster

Reply:
Just like in the UI5 Tooling...

https://github.com/SAP/ui5-cli/blob/master/README.md#L9

Reply:
good point